content/docs/user-guide/networking/multiplayer/split.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Client and Server Launchers with the Multiplayer Gem
+description: Build Open 3D Engine launchers with the Multiplayer Gem specifically targeting Clients, Servers or both.
+linkTitle: Separating Client and Server
+---
+
+The Multiplayer Gem supports builds that contain only client logic, only server logic, or both client and server logic. This allows users to create executables of smaller size by excluding unnecessary logic and dependencies. It also allows hiding potentially sensitive logic unique to one executable from the other. For example, a publicly released game client that doesn't ship with server logic can make hacking the simulation more difficult.
+
+The splitting functionality produces multiple build types.
+* GameLauncher is a client only launcher
+* ServerLauncher is a server only launcher suitable for Dedicated Servers
+* UnifiedLauncher provides both and is suitable for Client Hosted Servers
+
+This functionality is implemented through a variety of build mechanisms and it's important to understand these mechanisms in any Gem or project using the Multiplayer Gem.
+
+## Splitting Up Client and Server logic
+
+The Multiplayer Gem contains code files that can be divided into two categories.
+1. Files that are fully required on all launcher types.
+2. Files that have parts conditionally compiled out depending on launcher type and their dependents.
+
+These file lists are maintained in multiplayer_files.cmake and multiplayer_split_files.cmake respectively.
+
+multiplayer_files.cmake generally contains core datatypes, base and core classes. multiplayer_split_files.cmake contains AutoComponent based MultiplayerComponents and types dependent on them.
+
+### CMake setup
+
+The split by cmake files leads us to four Multiplayer targets.
+
+1. Common - A target containing multiplayer_files.cmake
+2. Client - A target containing multiplayer_files.cmake plus multiplayer_split_files.cmake conditionally compiled for Clients
+3. Server - A target containing multiplayer_files.cmake plus multiplayer_split_files.cmake conditionally compiled for Servers
+4. Unified - A target containing multiplayer_files.cmake plus multiplayer_split_files.cmake conditionally compiled for Clients and Servers
```

Comment:
Do we provide any guidance in how to ignore avoid generating a unified target? Imagine a lot use cases will be pure client/server split.

Reply:
The Unified target is needed in a few places (such as the Editor) so that's unfortunately not the case. The user is however under no obligation to build or use the UnifiedLauncher. The build target will exist but is in no way required.